### PR TITLE
Updated the plugin from npe1 to npe2

### PR DESCRIPTION
**Pull request by ch-n**
As the plugin had some issues with the newest napari version (0.4.14) I updated it to the new napari plugin engine 2.
Now everything works fine again and the plugin is ready for future napari releases.

### CODE REVIEW

**Comment by thewtex** [comment]
@ch-n wonderful, thanks for the update!

Any tips on updates for the CI configuration?

**Comment by ch-n** [comment]
I'm sorry but I'm not sure CL stands for.

**Comment by dzenanz** [comment]
Article about [CI](https://en.wikipedia.org/wiki/Continuous_integration). CI [script](https://github.com/InsightSoftwareConsortium/napari-itk-io/blob/main/.github/workflows/test_and_deploy.yml).

**Comment by ch-n** [comment]
Thanks for the clarification.
Did not take a look at the test_and_deploy script. There are some changes. I guess I shuld include them if nothing speaks against it.
Other then that I have no tips. I'm no expert in this regard.

File: .github/workflows/test_and_deploy.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, '3.10']
```
Comment (dzenanz) [inline .github/workflows/test_and_deploy.yml:26]: I guess that version number should not be quoted? But I might be wrong.

Comment (ch-n) [reply .github/workflows/test_and_deploy.yml:26]: Well I'm not sure my self but that's how it is in the current template for npe2 plugins.

**Review by dzenanz — APPROVED** [review]

File: .github/workflows/test_and_deploy.yml (hunk 1 above)
Comment (thewtex) [reply .github/workflows/test_and_deploy.yml:26]: It does need to be quoted to be interpreted correctly by GitHub Actions.

**Comment by thewtex** [comment]
> Did not take a look at the test_and_deploy script. There are some changes. I guess I should include them if nothing speaks against it.

@ch-n yes, wherever you found these updates, there are also likely updates to the `.github/workflow/*` files that we could include so the automated continuous integration (CI) testing checks will pass.

**Comment by ch-n** [comment]
Well I forgot to update the tox testing env in the tox.ini. Next try...

**Comment by ch-n** [comment]
I will close this pull request und try to build it from scratch hope it will work out...

**Pull request closed without merging**